src/components/App/App.js

```diff
@@ -1,22 +1,29 @@
 import './App.css'
 import LandingPage from '../LandingPage/LandingPage'
 import { Profile } from '../Profile/Profile'
-import { userData } from '../../UserDummyData'
+import { CurrentTours } from "../CurrentTours/CurrentTours"
+import { userData } from '../../mockdata/UserDummyData'
+import currentToursData from "../../mockdata/CurrentToursDummyData"
 import { useState } from 'react'
 
 function App() {
-  const [user, setUser] = useState(userData)
+  const [user, setUser] = useState(userData);
+  const [currentTours, setCurrentTours] = useState(currentToursData)
+  // const [currentTours, setCurrentTours] = useState<object | null>(currentToursData)
+  // types currently aren't working here since this isn't a tsx file but I think we want it to be
```

Comment:
Yeah, i'm interested to see how that will work. Also, i'm wondering when we get into it if the state of this will live in the LandingPage component as well as all the data for PastTours.